cucumber/api/features/support/authz_helpers.rb

```diff
@@ -33,6 +33,12 @@ def i_have_a_new_resource(kind, identifier)
 
     @resources[resource_id] = @current_resource
   end
+
+  def set_annotation_to_resource(name, value, resource = @current_resource)
+    unless resource.annotations.find { |a| a.values[:name] == name }
+      resource.add_annotation name: name, value: value
```

Comment:
add_annotations is a function that does not yet exist so wouldn't I be getting method not found error?

Reply:
but the function does exist.

Reply:
it was used in `authz_steps.rb`. i just moved the usage to the helper